internal/mode/static/nginx/config/maps.go

```diff
@@ -21,6 +21,65 @@ func executeMaps(conf dataplane.Configuration) []executeResult {
 	return []executeResult{result}
 }
 
+func executeStreamMaps(conf dataplane.Configuration) []executeResult {
+	maps := createStreamMaps(conf)
+
+	result := executeResult{
+		dest: streamConfigFile,
+		data: helpers.MustExecuteTemplate(mapsTemplate, maps),
+	}
+
+	return []executeResult{result}
+}
+
+func createStreamMaps(conf dataplane.Configuration) []http.Map {
+	maps := make([]http.Map, 0, len(conf.HTTPServers))
```

Comment:
I think I meant to type TLSPassthroughServers there. Currently the length of maps is bounded by that I think

Reply:
For maps is it better to use an upper bound as the capacity or initialize it to size 0?

Reply:
In this case, it is hard to predict user pattern.
By bounding to the number of TLSPassthroughServers, we potentially waste memory, because number of passthrough listeners can be 1 and TLSRoutes (TLSPassthroughServers) - many, including hundreds.
so, I'd rather avoid setting the capacity to the upper bound for this case.